packages/frontend/src/pages/share.vue

```diff
@@ -162,6 +162,12 @@ function goToMisskey(): void {
 	location.href = '/';
 }
 
+watch(state, (to) => {
+	if (to === 'posted') {
+		postMessage('misskey:shareForm:shareCompleted');
```

Comment:
グローバル明示する方が分かりやすくて好み
```suggestion
		window.postMessage('misskey:shareForm:shareCompleted');
```

Reply:
あーimportしてるのか

Reply:
🙏

Reply:
これ共通のラッパー関数で型定義当ててあります（@/script/post-message.ts）

Reply:
ただ確かにwindowのやつと混同する可能性はあるので名前変えたほうがいいかも